setup.py

```diff
@@ -38,6 +38,19 @@
           long_description=open('README.md').read(),
           long_description_content_type='text/markdown',
           python_requires='~=3.6',
+          install_requires=[
+              'mne >=0.21',
+              'numpy >=1.14',
+              'scipy >=0.18.1',
+          ],
```

Comment:
Is this also going to install these or upgrade? I personally don't like it when packages do this. It messes with the environment they have.

Reply:
If these requirements are not met, they will be installed.

If you don't like this, simply run `pip` with the `--no-deps` switch.